core/kazoo_attachments/src/kz_att_http.erl

```diff
@@ -139,11 +139,15 @@ format_url(Fields, JObj, Args, Separator) ->
 format_url_field(JObj, Args, Fields, Acc)
   when is_list(Fields) ->
     [format_url(Fields, JObj, Args, <<>>) | Acc];
+format_url_field(JObj, Args, #{<<"arg">> := Arg}, Fields) ->
+    format_url_field(JObj, Args, {arg, Arg}, Fields);
```

Comment:
ah, good point. `kz_att_http` and `kz_att_ftp` were coded to provide backward compatibility for existing deployments for usage in  `kz_media_record`. using them as "real" att handlers in the `data plans`  is cool :)
thanks